smart-contracts/helpers/bridge.js

```diff
@@ -0,0 +1,49 @@
+// see https://docs.connext.network/resources/deployments as referencce
+
+const mumbai = {
+  domainId : 9991,
+  connext : '0x2334937846Ab2A3FCE747b32587e1A1A2f6EEC5a',
+  testERC20: '0xeDb95D8037f769B72AAab41deeC92903A98C9E16',
+  testLockERC20: '0xcB145795F79DeA52b6c8F2d29cA1e81E7e99A0f2',
+  testLockNative: '0x501e0c8216d8f53ea92faf70b573bab9344f8038',
+  wethAddress : '0xFD2AB41e083c75085807c4A65C0A14FDD93d55A9',
+  bridgeAddress: '0x2334937846Ab2A3FCE747b32587e1A1A2f6EEC5a'
```

Comment:
Do we want to add these values in the networks package?

Reply:
most of these are test values for now but yes we will need a section with all bridge addresses and dao address

Reply:
as discussed, this will be addressed in a separate PR